install.ps1

```diff
@@ -3,7 +3,8 @@ $ErrorActionPreference = "Stop"
 # Install choco
 Set-ExecutionPolicy Bypass -Scope Process -Force
 iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
-
+$env:ChocolateyInstall = Convert-Path "$((Get-Command choco).path)\..\.."
+Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
 function Write-PackageInstall($package) {
```

Comment:
Please have a blank line above this to maintain readability.

Reply:
ok